App.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React from "react";
+import styled from "styled-components/native";
+import Quotes from "./components/Quotes";
+import { Text } from "react-native";
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
+  background-color: #c1e05b;
   justify-content: center;
   align-items: center;
-`
+`;
 
 const Title = styled.Text`
+  font-size: 34px;
+  font-weight: bold;
+  color: #ffffff;
+`;
+const Subtitle = styled.Text`
```

Comment:
super picky here but add a blank line between 17 and 18 :wink

Reply:
Thnx, will do :)